### PR TITLE
Set max tolerance for detectors on slither CI to "high"

### PR DESCRIPTION
**Pull request by RyanRHall**

### CODE REVIEW

File: src/v1/examples/ERC20Distributor.sol [hunk 1]
```diff
@@ -90,11 +91,12 @@ contract ERC20Distributor is MockERC20, LPNClientV1 {
...
+        lastClaimed[holder] = endBlock;
+
         uint256 requestId = lpnRegistry.request{value: lpnRegistry.gasFee()}(
             queryHash, placeholders, startBlock, endBlock
         );
 
-        lastClaimed[holder] = endBlock;
         requestData[requestId] =
             RequestData({holder: holder, blockRange: endBlock - startBlock + 1});
     }
```
Comment (RyanRHall) [inline src/v1/examples/ERC20Distributor.sol:102]: change is made to enforce the "checks-effects-interactions" patter (which fixes a reentrancy detection)

File: src/v1/examples/LayeredPenguins.sol [hunk 2]
```diff
@@ -48,6 +48,7 @@ contract LayeredPenguins is LPNClientV1, ERC721Enumerable {
         mintRequests[requestId] = MintRequest({sender: msg.sender});
     }
 
+    // slither-disable-next-line arbitrary-send-eth
```
Comment (RyanRHall) [inline src/v1/examples/LayeredPenguins.sol:51]: false positive

File: src/v1/examples/ERC20Distributor.sol [hunk 3]
```diff
@@ -60,6 +60,7 @@ contract ERC20Distributor is MockERC20, LPNClientV1 {
     /// @param placeholders_ Optionally allow placeholders to be overridden for demo purposes
     /// @param startBlock_ Optionally allow startBlock to be overridden for demo purposes
     /// @param endBlock_ Optionally allow endBlock to be overridden for demo purposes
+    // slither-disable-next-line arbitrary-send-eth
```
Comment (RyanRHall) [inline src/v1/examples/ERC20Distributor.sol:63]: false positive

File: src/v1/client/LPNQueryV1.sol [hunk 4]
```diff
@@ -59,13 +59,12 @@ contract LPNQueryV1 is LPNClientV1, Initializable {
...
-     * @param placeholders An array of placeholder values for the query.
-     * @param startBlock The starting block number for the query range.
-     * @param endBlock The ending block number for the query range.
-     */
+    /// @dev Function to submit a query to the Lagrange ZK Coprocessor.
+    /// @param queryHash The hash of the query to be executed.
+    /// @param placeholders An array of placeholder values for the query.
+    /// @param startBlock The starting block number for the query range.
+    /// @param endBlock The ending block number for the query range.
+    // slither-disable-next-line arbitrary-send-eth
```
Comment (RyanRHall) [inline src/v1/client/LPNQueryV1.sol:67]: false positive

**Review by kaisbaccour — APPROVED** [review]

**Pull request merged**